src/app/app-routing.module.ts

```diff
@@ -1,5 +1,7 @@
 import { NgModule } from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
+import { AuthenticateGuard } from './core/guards/authenticate.guard';
+import { UnauthenticatedGuard } from './core/guards/unauthenticated.guard';
```

Comment:
Why need two guards ? You can pass data by using 
```
{
    path: 'login',
    loadChildren: () => import('./modules/login/login.module').then((m) => m.LoginModule),
    canActivate: [UnauthenticatedGuard],
    data: { loggedIn: false }
  },
```
then use `route.data.props` inside AuthenticateGuard

https://stackoverflow.com/questions/42719445/pass-parameter-into-route-guard

Reply:
Oh